react/food-app/src/components/foodCard.test.jsx

```diff
@@ -1,23 +1,23 @@
-import {render, screen} from '@testing-library/react'
-import Foodcard from './foodCard'
-describe('Testing Food Card', () => {
-   describe('Testing Label', () => {
-    it('Shows Label Title', () => {
-        render(<Foodcard/>)
-        const resultElement = screen.getByText('Search For Your Favourite Food')
-        expect(resultElement).toBeInTheDocument()
-    });
-   });
-   describe('Testing Each Food Card', () => {
-       it('renders each card', () => {
-           render(<Foodcard/>)
-           const resultElementId = screen.getByTestId('food-card-item')
-           expect(resultElementId).toBeInTheDocument()
-       });
+// import {render, screen} from '@testing-library/react'
+// import Foodcard from './foodCard'
+// describe('Testing Food Card', () => {
```

Comment:
If this test isn't working anymore it would be preferable to `.skip` it or just remove it if you're no longer happy with it. Committing commented out code that you think you might want later isn't necessary because you can just delete it and recover it from the repository's history later.

Reply:
Okay will change that, 
Thank you